src/main/java/Database/playlistLibrary.java

```diff
@@ -124,7 +124,9 @@ public playlistDsData findPlaylist(int id) {
             return new playlistDsData(p);
         }
 
-        return playlistDatabase.get(id);
```

Comment:
What you had originally here was fine -- hashmap's default return value if id is not in the keys is null

Reply:
Not sure if that’s true, I was getting a NullPointerException trying to get playlists that didn’t exist

Reply:
Damn, really? If I remember right my implementation of SongLibrary relied on the null return and from what I saw (via good ol print checks) it /was/ indeed returning them. That's weird

Reply:
It could just be me being stupid lol but this change isn’t breaking anything so might as well just leave it.